feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/MoodEvent.java

```diff
@@ -65,6 +78,9 @@ public String getReason() {
     }
 
     public void setReason(String reason) {
+        if(reason.length()>MAX_REASON_LEN){
+            throw new IllegalArgumentException("The reason is longer than "+MAX_REASON_LEN+" characters!");
```

Comment:
my pythonic ass is screaming at the concatenation of a string and an int but if this works please ignore me

Reply:
This is acceptable in java. toString is called when this occurs